Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -972,14 +974,31 @@ def build_drilldown_search(notable_data, search, raw_dict, is_query_name=False):
             if not is_query_name:
                 demisto.error(f'Failed building drilldown search query. Field {raw_field} was not found in the notable.')
             return ""
+
         if prefix:
-            replacement = get_fields_query_part(notable_data, prefix, [field], raw_dict)
+            demisto.debug(f"field is User and Prefix - {field}")
+            if field in USER_RELATED_FIELDS:
+                add_backslash = True
```

Comment:
why this variable "add_backslash"  is needed and not just pass true if we enter this condition?
like: 
eplacement = get_fields_query_part(notable_data, prefix, [field], raw_dict, true)

Reply:
@yuvalbenshalom, I added it just for readability. I thought it would be clearer that way. 
But you are right, it is redundant, will change that.